airbyte-integrations/connectors/destination-dynamodb/src/test/java/io/airbyte/integrations/destination/dynamodb/DynamodbDestinationStrictEncryptTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.dynamodb;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.protocol.models.AirbyteConnectionStatus;
+import org.junit.jupiter.api.Test;
+
+public class DynamodbDestinationStrictEncryptTest {
+
+  private static final String NON_SECURE_URL_ERR_MSG = "Server Endpoint requires HTTPS";
```

Comment:
Unless I'm mistaken you can probably reuse the `NON_SECURE_URL_ERR_MSG` variable by making it `protected` in `DynamodbDestinationStrictEncrypt`, this creates a single source of truth so code doesn't deviate over time and easier to maintain

Reply:
Updated, thanks